documentation/asciidoc/topics/proc_adding_credentials_keystore.adoc

```diff
@@ -0,0 +1,24 @@
+[id='adding-credentials_keystore_{context}']
+= Adding credentials keystore
```

Comment:
Should we make this more broad and just define how to create credentials in general, as we reference this section in the specific Keystore/Truststore documentation context.

So maybe just "Adding Credentials Store" for the heading and then update the steps to not explicitly reference Keystores?

Reply:
yeah sounds good